src/views/start/index.jsx

```diff
@@ -121,6 +122,18 @@ export default class Start extends Component {
     this.fileExtension = e.target.value;
   }
 
+  onFileExtensionExclusionKeyDown = (e) => {
+    if (e.keyCode === 13) {
+      if (this.currentPath) {
+        gLog(this.currentPath, this.gLogDoneCB, this.queryValue, this.fileExtension, this.fileExtensionExclusion);
+      }
+    }
+  }
+
+  setFileExtensionExclusionValue = (e) => {
+    this.fileExtensionExclusion = e.target.value;
```

Comment:
Reminder for me: is this logging a warning? Controlled uncontrolled input? Check that -> @L4B0MB4  :p